kogito-editors-js/packages/boxed-expression-component/src/__tests__/components/FunctionExpression/FunctionExpression.test.tsx

```diff
@@ -248,14 +249,14 @@ describe("FunctionExpression tests", () => {
 
     function checkFormalParameters(mockedBroadcastDefinition: jest.Mock, formalParameters: EntryInfo[]) {
       expect(mockedBroadcastDefinition).toHaveBeenCalledWith({
-        dataType: DataType.Undefined,
+        dataType: undefined,
```

Comment:
Why are we changing the expectations of this test case?

Reply:
I've changed this because I thought the test case changed, but I was wrong. Actually, it was a bug on the FunctionExpression. I've reverted.